Issue 160

constellates opened the pull request:
Fix for issue #160

chrisfilo commented:
This will still give an effect of the same file being listed twice in the non-verbose view. Maybe the best option would be to split it into two errors with separate codes? @wtriplett

constellates commented:
What's a case where this would still list the same file twice within that issue? Previously if both EchoTime1 and EchoTime2 were missing it would check those individually and add the file twice. I updated the check so that if either or both EchoTime1 and EchoTime2 are missing it will add the issue/file.  

I've tested it and switched back and forth to master to verify the issue and it seems to be working as expect, but I've only tested a couple limited cases.

chrisfilo commented:
Sorry - you are right! I misread the code!
